processes/code-styles/swift/swift-style-guide-en.md

```diff
@@ -1,59 +1,269 @@
 # Rambler Swift Style Guide
```

Comment:
Check for compliance with https://tools.ietf.org/html/rfc2119

Reply:
At least this document SHOULD incorporate this words

> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
>       NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
>       "OPTIONAL" in this document are to be interpreted as described in
>       RFC 2119.

Reply:
you mean what I SHOULD write such words in the document in ALL CAPITALS where needed?